src/HomeWork.java

```diff
@@ -0,0 +1,91 @@
+
+public class HomeWork {
+    public static int calculateAbsoluteDifference(int a, int b) {
+        return Math.abs(a - b);
+    }
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+        int result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+        a = -8;
+        b = 7;
+        result = calculateAbsoluteDifference(a, b);
+
+        System.out.println("a = " + a);
+        System.out.println("a = " + b);
+        System.out.println("result = " + result);
+
+    }
+}
+
+class FirstChar {
+    public static char getFirstCharacter(String inputString) {
+        if (inputString != null && !inputString.isEmpty()) {
+            char[] charArray = inputString.toCharArray();
+            return charArray[0];
+        } else {
+            throw new IllegalArgumentException(" ");
+        }
+    }
+
+    public static void main(String[] args) {
+        String string = "Python";
+        char result = getFirstCharacter(string);
+        System.out.println("string = \"" + string + "\"");
+        System.out.println("result = '" + result + "'");
+    }
+}
+
+class LastChar {
```

Comment:
Same about class and method. Solution is overall correct, do the first task like this